packages/tocco-ui/src/Icon/mapping.json

```diff
@@ -337,6 +337,41 @@
       "trash"
     ]
   },
+  "comments": {
```

Comment:
i think its important to support a clear icon set. right now it seems that every other icons is getting a light version as well. Can't we use one variation everywhere? Or dont you think a clear icon set is needed for clarity and consistency?